tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/client/StoreConfigTest.java

```diff
@@ -289,6 +294,107 @@ public void testCreateQueue() throws Exception {
       jmsServer.stop();
    }
 
+   @Test
+   public void testCompatibilityWith260() {
+      List<String> transportConfigurations = new ArrayList<>();
+      transportConfigurations.add("tst");
+      ConnectionFactoryConfigurationImpl configuration = (ConnectionFactoryConfigurationImpl) new ConnectionFactoryConfigurationImpl();
+      configuration.setName("np").setConnectorNames(transportConfigurations);
+
+      ByteBuffer buffer = ByteBuffer.allocate(configuration.getEncodeSize());
+      ActiveMQBuffer activeMQBuffer = new ChannelBufferWrapper(Unpooled.wrappedBuffer(buffer));
+      activeMQBuffer.clear();
+      encodeVersion260(activeMQBuffer, configuration);
+      configuration.decode(activeMQBuffer);
+   }
+
+   public void encodeVersion260(final ActiveMQBuffer buffer, final ConnectionFactoryConfigurationImpl connectionFactoryConfiguration) {
```

Comment:
I think you should use the compatibility tests for this instead. disagree with the test itself here.

Reply:
It was to just demo the introduced bug by recent merged pr. I can simply remove and leave the fix

Reply:
@michaelandrepearce merged... removed your test.. added mine.. take a look please.

Reply:
Already noticed, Looks fine and dandy. Probably worth adding another version at some point maybe after 2.6.3 to the versions to be compatibility checked, just as lots of recent serialisation changes and additions